src/plugins/expressions/common/expression_functions/series_calculation_helpers.ts

```diff
@@ -44,9 +44,13 @@ export function buildResultColumns(
   input: Datatable,
   outputColumnId: string,
   inputColumnId: string,
-  outputColumnName: string | undefined
+  outputColumnName: string | undefined,
+  options: { allowColumnOverride: boolean } = { allowColumnOverride: false }
 ) {
-  if (input.columns.some((column) => column.id === outputColumnId)) {
+  if (
+    !options.allowColumnOverride &&
```

Comment:
naming nit: I think `allowColumnOverwrite` would be a bit more intuitive than "Override", but I don't feel strongly about this.